modules/aruco/src/aruco.cpp

```diff
@@ -606,7 +658,7 @@ static uint8_t _identifyOneCandidate(const Ptr<Dictionary>& dictionary, InputArr
     Mat onlyBits =
         candidateBits.rowRange(params->markerBorderBits,
                                candidateBits.rows - params->markerBorderBits)
-            .colRange(params->markerBorderBits, candidateBits.cols - params->markerBorderBits);
+            .colRange(params->markerBorderBits, candidateBits.rows - params->markerBorderBits);
```

Comment:
looks strange.

Reply:
yes)
This is from this PR #2906 https://github.com/opencv/opencv_contrib/pull/2906
PR looks strange too)))

Reply:
I will fix it